### ISSUE TITLE
requirements/dev_python*.txt broken on 2018.3 and develop

### ISSUE THREAD

**Issue by sergeizv** [saltstack/salt#47542]
### Description of Issue/Question

requirements/dev_python27.txt and requirements/dev_python34.txt on 2018.3 and develop branches include base-py2.txt and base-py3.txt files which don't exist.

### Setup
salt git repo with develop or 2018.3 branch checked out.

### Steps to Reproduce Issue

```
$ cd salt
$ git checkout 2018.3
$ pip install -r requirements/dev_python27.txt 
Could not open requirements file: [Errno 2] No such file or directory: 'requirements/base-py2.txt'
```

### Versions Report
(Provided by running `salt --versions-report`. Please also mention any differences in master/minion versions.)

**Comment by sergeizv** [comment]
The references to missing files were introduced in  #47374

**Comment by sergeizv** [comment]
I apologize for occasionally changing the status of this issue...

**Comment by Ch3LL** [comment]
fixed by https://github.com/saltstack/salt/pull/47433  will be in the next merge forward to 2018.3 then develop

**Comment by sergeizv** [comment]
On Tue, May 08, 2018 at 04:09:23PM +0000, Megan Wilhite wrote:
> fixed by https://github.com/saltstack/salt/pull/47433  will be in the next merge forward to 2018.3 then develop
> 
> -- 
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/saltstack/salt/issues/47542#issuecomment-387455970

Thank you! Didn't see that PR :)

**Comment by Ch3LL** [comment]
no problem :) was merged into 2018.3 in this merge forward https://github.com/saltstack/salt/pull/47514 will go ahead and close thanks
